docs/REST API/rest-api.md

```diff
@@ -4584,6 +4592,33 @@ For example, a policy might ask “Is Gatekeeper enabled on macOS devices?“ Th
 }
 ```
 
+---
+
+### Count Policies
```

Comment:
Convention is to sentence-case titles (side note: if you ever have doubts about what to treat as a proper noun in Fleet, a lot of that stuff is listed out [here](https://fleetdm.com/handbook/company/communications#sentence-case) — I know there can be ambiguity around feature names & stuff)
```suggestion
### Count policies
```